tests/msim/insert_keyword.py

```diff
@@ -0,0 +1,15 @@
+def run(ecl_state, schedule, report_step, summary_state):
+	print("current report_step: {}".format(report_step))
+	if (report_step == 2):
```

Comment:
If people want to do this,  then we probably need a means to do this only once or to check whether this is the first time this executed in the report step..

Like this we would insert the keywords multiple times if there are multiple timesteps during report step 2.

Reply:
Somehow this already possible with the parameters to PYACTION. Might need many PYACTION keywords, though.

Reply:
Yes, definitely - let's discuss later today?

Reply:
I've moved the insert_keyword things to another PR